tests/lib/rules/key-spacing.js

```diff
@@ -1769,5 +1977,218 @@ ruleTester.run("key-spacing", rule, {
             { messageId: "missingKey", data: { computed: "", key: "foo" }, line: 1, column: 7, type: "Identifier" },
             { messageId: "missingValue", data: { computed: "", key: "foo" }, line: 1, column: 19, type: "Identifier" }
         ]
+    },
+    {
+        code: [
+            "({",
+            "    foo : 1, bar : 2, baz : 3",
+            "});"
+        ].join("\n"),
+        output: [
+            "({",
+            "    foo: 1, bar: 2, baz: 3",
+            "});"
+        ].join("\n"),
+        options: [{
+            beforeColon: false
+        }],
+        errors: [
+            { messageId: "extraKey", data: { computed: "", key: "foo" }, line: 2, column: 5, type: "Identifier" },
```

Comment:
unrelated: it would be better to refactor the messageIds to `"extraKey" || "extraComputedKey"`, rather than passing `data: {"computed": ""}`